catalog-info.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: "-shared-infrastructure"
```

Think this needs changed 👀

```suggestion
  name: "ecm-shared-infrastructure"
```